LCD.md

```diff
@@ -0,0 +1,19 @@
+# LCD screen and touch controllers list
```

Comment:
Could we add a reference to this file (a symlink?) to `componets/lcd` and `components/lcd_touch`? What do you think?

Reply:
Symlink would it be nice, but I don't know, if it will work on windows right. I tried but cannot make it. Other way can be create README file for each folder and there add section with LCD list.

Reply:
Creating a README file with a few sentences and a link to LCD readme sounds like a good solution.

Reply:
Added README files into lcd and lcd_touch folders.